### ISSUE TITLE
Opening link in about menu with keyboard navigation opens two tabs

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/molarity#222]
For https://github.com/phetsims/QA/issues/476. I noticed that opening a link in the about menu with keyboard navigation opens two tabs. I've seen this in macOS + Chrome and macOS + Safari. I tried to replicate this with John Travoltage, BASE, and Friction on the website, but I wasn't able to, hence posting this issue in the Molarity repository.

This doesn't work with the "Report a Problem".

**Comment by jessegreenberg** [comment]
Confirmed.

**Comment by jessegreenberg** [comment]
Ah, I am also seeing this on Windows machines too.

**Comment by jessegreenberg** [comment]
@lmulhall-phet can you please test master after the above commit and make sure this is fixed?

**Comment by lmulhall-phet** [comment]
I just pulled on phettest, but it doesn't seem to be fixed on macOS + Chrome.

**Comment by jessegreenberg** [comment]
Sorry @lmulhall-phet I may have assigned to you too soon. I see the scenery repo is out of date on phettest, can you try pulling that and checking again?

**Comment by lmulhall-phet** [comment]
Ah, I feel stupid. Seems to be fixed now. Thanks, @jessegreenberg.

**Comment by jessegreenberg** [comment]
Excellent, thank you. Ill cherry-pick the sha to the scenery branch for the next rc.

**Comment by jessegreenberg** [comment]
Cherry-picked to scenery branch and dependencies.json updated to capture the SHA.

**Comment by KatieWoe** [comment]
Looks good in rc4. Tested on Mac 10.15 chrome and safari

**Comment by KatieWoe** [comment]
Seeing this in https://github.com/phetsims/QA/issues/480

**Comment by zepumph** [comment]
I need to cherry-pick https://github.com/phetsims/scenery/commit/876f3af68cea2e2ad2755e3ea5048ceab90d1eb2 to the GFL 2.2 branch.

**Comment by KatieWoe** [comment]
Looks good in rc3
